themes/scss/widgets/text-elements.scss

```diff
@@ -12,16 +9,16 @@
 .CDB-Widget-textBig { // 16 size
   @include text-overflow();
   display: inline-block;
-  font-size: $sFontSize-larger;
+  font-size: $sFontSize-large;
   font-weight: $sFontWeight-normal;
-  line-height: $sFontSize-normal + 6;
+  line-height: $sFontSize-medium + 6;
```

Comment:
Name of variable `sFontSize-medium` should be written in all lowercase letters with hyphens instead of underscores